pkg/nad/masterplugin.go

```diff
@@ -393,3 +393,65 @@ func (plugin *MasterIPVlanPlugin) GetMasterPluginConfig() (*MasterPlugin, error)
 
 	return plugin.masterPlugin, nil
 }
+
+// MasterBoundPlugin provides struct for MasterPlugin to create a NeNetworkAttachmentDefinition Master plugin for bonded interfaces.
```

Comment:
Typo? MasterBondPlugin

Reply:
Changed all references to MasterBondPlugin